clients/drcachesim/tools/filter/record_filter.cpp

```diff
@@ -95,7 +95,7 @@ parse_string(const std::string &s, char sep = ',')
     std::vector<T> vec;
     do {
         pos = s.find(sep, at);
-        vec.push_back(static_cast<T>(std::stoi(s.substr(at, pos))));
+        vec.push_back(static_cast<T>(std::stoll(s.substr(at, pos))));
```

Comment:
This makes sense only when T is a 64-bit int. Maybe add a check that T can hold the parsed value.

Reply:
Done.
If T cannot hold the value, we skip it.
Probably not the best way to handle it, but I guess it's slightly better than before (when we would simply truncate the value and add it to the vector anyway).
Happy to go with a better option if you have one in mind.